Kajabi/sage-lib
# [SAGE-657] Update Charcoal 500 color token

## Description
Modifies the style dictionary entry for `charcoal 500` to match the value in Figma.

Also updates the Sage docs Colors page to remove deprecated values and improve readability.

## Screenshots

|  Before  |  After  |
|--------|--------|
|(no output)|<img src="https://user-images.githubusercontent.com/816579/170573627-5ea3c4ea-f9ee-427a-8f20-5b235646e31b.png" alt="" width="240">|


## Testing in `sage-lib`
1. Navigate to the [Foundations->Color page](http://localhost:4000/pages/foundations/color?sage_theme=sage_theme_next) in your local Sage docs
2. Verify that the list of `charcoal` colors appears
3. Confirm that `charcoal 500` matches the expected hex value `#040506`
4. (optional) Navigate to the [Nav Link component page](http://localhost:4000/pages/component/nav_link?tab=preview), and inspect any of the link examples
5. (optional) Confirm that the link's `charcoal 500` value matches the expected hex `#040506`


## Testing in `kajabi-products`

1. (**HIGH**) Will impact all uses of `charcoal-500` within the app when `sage_next` feature flag is enabled.


## Related
- https://github.com/Kajabi/sage-lib/pull/1420
